tests/measurements/test_purity_measurement.py

```diff
@@ -323,52 +323,3 @@ def circuit(x):
         grad_purity = tape.gradient(purity, param)
 
         assert qml.math.allclose(grad_purity, grad_expected_purity, rtol=1e-04, atol=1e-05)
-
-    @pytest.mark.parametrize("device", devices)
-    @pytest.mark.parametrize("return_new", return_new)
-    def test_qnode_entropy_no_custom_wires(self, device, return_new):
-        """Test that purity cannot be returned with custom wires."""
-
-        if return_new:
-            qml.enable_return()
-
-        dev = qml.device(device, wires=["a", 1])
-
-        @qml.qnode(dev)
-        def circuit(x):
-            qml.IsingXX(x, wires=["a", 1])
-            return qml.purity(wires=["a"])
-
-        with pytest.raises(
-            qml.QuantumFunctionError,
-            match="Returning the purity is not supported when using custom wire labels",
-        ):
-            circuit(0.1)
-
-        if return_new:
-            qml.disable_return()
```

Comment:
Instead of removing this, it would be nice to test that `purity` works with custom wires.